test/typescript/init-with-colors/test.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Testing interaction with Terraform Cloud
+ *
+ * @group typescript
+ */
+
+import { TestDriver } from "../../test-helper";
+
+// Since chalk auto-detects the capabilities, colored output is
+// deactivated by default in non-tty environments. We had an issue
+// in the past, where this turned out to be a problem
+
+describe("test with colors", () => {
+  let driver: TestDriver;
+
+  test("build providers", () => {
```

Comment:
I know this matches what was there, but is it really testing anything?

Reply:
Just looked it up again - that was the original issue why this test is here https://github.com/hashicorp/terraform-cdk/issues/141 - There's probably a better way to test this though. Gonna think about it.

Reply:
Added a reference to the test. I think it would be better to test this in isolation, but didn't want to get into this right now :) f2fbc79